lib/src/main/kotlin/com/swmansion/starknet/account/StandardAccount.kt

```diff
@@ -48,10 +48,7 @@ class StandardAccount(
 
     override fun signV1(calls: List<Call>, params: ExecutionParams, forFeeEstimate: Boolean): InvokeTransactionV1Payload {
         val calldata = AccountCalldataTransformer.callsToExecuteCalldata(calls, cairoVersion)
-        val signVersion = when (forFeeEstimate) {
-            true -> estimateVersion(Felt.ONE)
-            false -> Felt.ONE
-        }
+        val signVersion = if (forFeeEstimate) TransactionVersion.V1_QUERY else TransactionVersion.V1
```

Comment:
Maybe we could change the name `v1_query` to something more meaningful? I guess this version can be used only for estimates, right? so. perhaps something connected to it? (unless this is how it's called in spec or something, then 🤷 )

Reply:
It's not called that way in the spec, but it is always referred to as "query version". We can of course rename it to `V1_FOR_FEE_ESTIMATE`, but it might be a bit misleading since it's also used in `simulateTransactions`.

Maybe we should instead write a description of the enum for the docs, explaining the purpose of query versions?

Reply:
> Maybe we should instead write a description of the enum for the docs, explaining the purpose of query versions?

yes, let's